### ISSUE TITLE
Plugin crashed at the end of exporting to Avocode

### ISSUE THREAD

**Issue by marco-uccellini** [avocode/avocode#632]
IMPORTANT: Please try to describe how did it happen.
This occurred when updating a project. The plugin worked perfectly until it got to the end stage "Updating Layers" at wich point it crashed.

-----
Sketch Exporter 2.1.5 running in Sketch 3.5
```
Error: Error: Command failed: 2016-01-27 15:46:18.443 sketchtool[15499:4199281] Error: caught exception whilst suppressing update events: *** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer)
2016-01-27 15:46:18.444 sketchtool[15499:4199281] Error: caught exception whilst decoding (version 76):*** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer)
Failed to open document file:///Users/marco/0-Projects/Alan%20Buckett/icephorm/Sketch/icephorm.sketch
((null) MSDocumentDomain:11)


    at /Users/marco/…/avocode-exporter/lib/image-exporter.js:1:873
    at ChildProcess.exithandler (child_process.js:651:7)
    at ChildProcess.EventEmitter.emit (events.js:98:17)
    at maybeClose (child_process.js:753:16)
    at Socket.<anonymous> (child_process.js:966:11)
    at Socket.EventEmitter.emit (events.js:95:17)
    at Pipe.close (net.js:465:12)
```

**Comment by rychlis** [comment]
We have just released Sketch Extension 3.0.0-prerelease4 with Sketch 3.5 support. Please download the latest version from https://github.com/avocode/avocode/blob/master/sketch-extension-beta.md#installation--update

We're working on updating the older Sketch Plugin 2 however there has been some further issues with Sketch 3.5 unfortunately. You should be able to use the beta version fine though

Follow #609 for more updates about older Sketch plugin, I am closing this issue as a duplicate, thanks for the report

**Comment by marco-uccellini** [comment]
Thanks for the fast response!
